pyproject.toml

```diff
@@ -60,7 +60,7 @@ dev = [
     "nbsphinx",
 ]
 publishing = ["build", "twine", "wheel"]
-tutorials = ["jupyter"]
+tutorials = ["notebook"]
```

Comment:
`build` is also part of the optional dev dependencies, but I don't think it needs to be there (just here for the publishing part).

Reply:
In the `build.yml` file we verify that we can build the package and we use `build`, that's why it's also in the `dev` list of deps

Reply:
we actually use both dependency lists there anyway. Not a huge issue, just seems odd to state the same package twice:

https://github.com/ANNUBS/neurogym/blob/83792a3634ef9428828afd88e8fa8a2a4890f295/.github/workflows/build.yml#L54

Reply:
oh yes you're right, I've just removed `build` from the `dev` list